File: Java/src/com/leetcode/Easy/SquaresOfASortedArray.java
package com.leetcode.Easy;

import java.util.*;

// 977. Squares of a Sorted Array
public class SquaresOfASortedArray {

    public int[] sortedSquares(int[] A) {
        for (int i = 0; i < A.length; i++) {
            A[i] = A[i] * A[i];
        }

        Arrays.sort(A);
        return A;
    }
}

File: JavaScript/Easy/SquaresOfASortedArray.js
/**
 * 977. Squares of a Sorted Array
 * @param {number[]} nums
 * @return {number[]}
 */
const sortedSquares = function (nums) {
  let result = [];
  nums.forEach(num => {
    result.push(num * num);
  });

  return result.sort((a, b) => a - b);
};